pkg/tsdbctl/info.go

```diff
@@ -41,16 +41,21 @@ func newInfoCommandeer(rootCommandeer *RootCommandeer) *infoCommandeer {
 
 	cmd := &cobra.Command{
 		Use:   "info",
-		Short: "return TSDB config",
+		Short: "Return information about a TSDB instance",
```

Comment:
Done — commit e1053ad — but I used "Display" and not "Print"; I also edited the flag descriptions accordingly (I also was didn't really like "Return", but I assumed you liked it because it was used in the existing help text so I decided to leave it.)
Note that the convention of the TSDB CLI help text is to phrase the description as what the command should do and now what it does; this was the existing convention and I didn't change it. So it should be "Print"/"Display" and not "Prints"/"Displays".